fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1407,6 +1484,10 @@ ACTOR Future<int> setDDMode( Database cx, int mode ) {
 				BinaryWriter wrMyOwner(Unversioned());
 				wrMyOwner << dataDistributionModeLock;
 				tr.set( moveKeysLockOwnerKey, wrMyOwner.toValue() );
+			} else {
+				// set DDMode to 1 will enable all disabled parts, for instance the SS failure monitors.
+				tr.clear(healthyZoneKey);
```

Comment:
It was intended until you asked. After some thought I will limit this to only clear it if it's the `ignoreSSFailures` value so that old maintenance behavior keeps as is.